app/src/main/java/ch/epfl/sweng/SDP/auth/LoginActivity.java

```diff
@@ -74,8 +75,15 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 assert response != null;
                 handleSuccessfulSignIn(response);
             } else {
+                // User pressed the back button
```

Comment:
In theory the user cannot press the back button anymore, but maybe it's better to leave this check for null